site/app/entities/poll/Poll.php

```diff
@@ -91,39 +103,52 @@ public function getQuestion(): string {
     public function setQuestion(string $question): void {
         $this->question = $question;
     }
-
-    public function getStatus(): string {
-        return $this->status;
+    public function setClosed(): void {
+        $this->end_date = new \DateTime(DateUtils::BEGINING_OF_TIME);
     }
-
     public function setOpen(): void {
-        $this->status = "open";
+        $this->end_date = new \DateTime(DateUtils::MAX_TIME);
+    }
+    public function setEnded(): void {
+        $temp = DateUtils::getDateTimeNow();
+        $tempString = $temp->format('Y-m-d');
+        $this->end_date = new DateTime($tempString);
```

Comment:
Is there a reason why all of this logic is necessary?  Why can't you just use `$this->end_date = DateUtils::getDateTimeNow();` directly?

Reply:
This method is only used for the test. I need it to be in Y - m - d format because I am checking for specifically that and have set the "expected output" in the test to match this format. It wouldn't work with hours minutes and seconds because those could deviate.